### ISSUE TITLE
Incompatiblity with linux-musl due to usage of statvfs64

### ISSUE THREAD

**Issue by ayakael** [djberg96/sys-filesystem#66]
The next version of Alpine Linux drops the alias for statvfs64 due to musl dropping that alias. Thus, sys-filesystem, since version 1.3.3, ceases to work.

**Comment by djberg96** [comment]
Hm, I'm guessing we could just put a rescue block around the definition like so:

```ruby

if linux64? || solaris?
  begin
    attach_function(:statvfs, :statvfs64, %i[string pointer], :int)
    attach_function(:statvfs, %i[string pointer], :int)
  rescue FFI::NotFoundError
    attach_function(:statvfs, %i[string pointer], :int)
  end
else
  attach_function(:statvfs, %i[string pointer], :int)
end
```

**Comment by djberg96** [comment]
Fixed in 1.4.5.
